docs/install_guides/installing-openvino-apt.md

```diff
@@ -1,36 +1,38 @@
 # Install Intel® Distribution of OpenVINO™ Toolkit for Linux Using APT Repository {#openvino_docs_install_guides_installing_openvino_apt}
 
-
-
 @sphinxdirective
 
 .. meta::
-   :description: Learn how to install OpenVINO™ Runtime on Linux operating 
-                 system, using the APT repository, which is a recommended 
-                 installation method for C++ developers.
-
-
-With the OpenVINO™ 2023.0 release, you can install OpenVINO Runtime on Linux using the APT repository. 
-OpenVINO™ Development Tools can be installed via PyPI only.
-See `Installing Additional Components <step-3-optional-install-additional-components>`__ for more information. 
-
-See the `Release Notes <https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino/2023-0.html>`__ 
-for more information on updates in the latest release.
+   :description: Learn how to install OpenVINO™ Runtime on the Linux operating 
+                 system, using the APT repository.
 
-Installing OpenVINO Runtime from APT is recommended for C++ developers. If you are working with Python, 
-the PyPI package has everything needed for Python development and deployment on CPU and GPUs. Visit the 
-:doc:`Install OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>` page for instructions 
-on how to install OpenVINO Runtime for Python using PyPI.
+.. note::
+   
+   Note that the APT distribution:
 
-.. warning:: 
+   * offers both C++ and Python APIs
+   * does not offer support for GNA and NPU inference
+   * additionally includes code samples 
+   * is dedicated to Linux users (may offer different hardware support under different 
+     operating systems, see the drop-down below for more details).
 
-   By downloading and using this container and the included software, you agree to the terms and 
-   conditions of the 
-   `software license agreements <https://software.intel.com/content/dam/develop/external/us/en/documents/intel-openvino-license-agreements.pdf>`__.
+   .. dropdown:: Inference Options 
 
+      ===================  =====  =====  =====  =====  ========  =============  ========  ========
```

Comment:
For APT we activating only RPMs/DEBs from following OSes:
Ubuntu18, Ubuntu20, Ubuntu22, RHEL8